### PR TITLE
Tiny js world

### PR DESCRIPTION
**Pull request by mustbefail**
### Tiny JS world

[Demo](https://mustbefail.github.io/a-tiny-JS-world/) |
[Code base](https://github.com/mustbefail/frontend-2021-homeworks/tree/tiny-js-world/submissions/mustbefail/tiny-js-world)

The code is submitted in a dedicated feature branch.
Please, review.

### CODE REVIEW

**Review by madmaxWMFU — APPROVED** [review]
LGTM👍🏼

File: submissions/mustbefail/tiny-js-world/index.js [hunk 1]
```diff
@@ -0,0 +1,89 @@
...
+  Code repository: _put repo URL here_
+  Web app: _put project's github pages URL here_
+  */
+
+// ======== OBJECTS DEFINITIONS ========
+// Define your objects here
+const cat = {
+  species: 'cat',
+  name: 'Prokhor',
+  hands: 0,
```
Comment (babayK0) [inline submissions/mustbefail/tiny-js-world/index.js:13]: Objects of species that do not have hands by natural design do not consequently have hands or any equivalent property

File: submissions/mustbefail/tiny-js-world/index.js [hunk 2]
```diff
@@ -0,0 +1,89 @@
...
+  saying: 'Meow!',
+  gender: 'male',
+  friends: 'June, Maki',
+};
+const catWoman = {
+  species: 'cat',
+  name: 'June',
+  hands: 0,
+  legs: 4,
+  saying: cat.saying,
```
Comment (babayK0) [inline submissions/mustbefail/tiny-js-world/index.js:24]: What will be if `cat` obj will be declared after `catWoman` ?

**Review by babayK0 — CHANGES_REQUESTED** [review]
Good job, see my comments below :cat:

File: submissions/mustbefail/tiny-js-world/index.js (hunk 2 above)
Comment (mustbefail) [reply submissions/mustbefail/tiny-js-world/index.js:24]: Error calling before initialization. Need to move a property **saying** to a shared object?

File: submissions/mustbefail/tiny-js-world/index.js (hunk 1 above)
Comment (mustbefail) [reply submissions/mustbefail/tiny-js-world/index.js]: Removed property hands

File: submissions/mustbefail/tiny-js-world/index.js (hunk 2 above)
Comment (babayK0) [reply submissions/mustbefail/tiny-js-world/index.js]: No, shared obj is not necessary here

Comment (babayK0) [reply submissions/mustbefail/tiny-js-world/index.js]: BTW, your idea is good. This is not a blocker so i can approve this pr

**Review by babayK0 — APPROVED** [review]

**Pull request merged**